Makefile

```diff
@@ -27,10 +31,30 @@ patch:
 	find ${SPEC_PATCHED_FILE} -empty -exec cp ${SPEC_FETCHED_FILE} ${SPEC_PATCHED_FILE} \;
 
 gen:
-	${SWAGGER} generate client \
+	${SWAGGER} generate -g go \
+		--package-name ${PACKAGE_MAJOR} \
 		--model-package types \
-		--additional-initialism bgp \
-		--additional-initialism vpn \
-		--additional-initialism vlan \
-		--additional-initialism vlans \
-		-f ${SPEC_PATCHED_FILE}
+		--api-package models \
+		--git-user-id ${GIT_ORG} \
+		--git-repo-id ${GIT_REPO} \
+		-o /local/${PACKAGE_MAJOR} \
+		-i /local/${SPEC_PATCHED_FILE}
+
+# https://github.com/OpenAPITools/openapi-generator/issues/741#issuecomment-569791780
+remove-dupe-requests: ## Removes duplicate Request structs from the generated code
+	@for struct in $$(grep -h 'type .\{1,\} struct' $(PACKAGE_MAJOR)/*.go | grep Request  | sort | uniq -c | grep -v '^      1' | awk '{print $$3}'); do \
+	  for f in $$(/bin/ls $(PACKAGE_MAJOR)/*.go); do \
+	    if grep -qF "type $${struct} struct" "$${f}"; then \
+	      if eval "test -z \$${$${struct}}"; then \
+	        echo "skipping first appearance of $${struct} in file $${f}"; \
+	        eval "export $${struct}=1"; \
+	      else \
+	        echo "removing dupe $${struct} from file $${f}"; \
+	        tr '\n' '\r' <"$${f}" | sed 's~// '"$${struct}"'.\{1,\}type '"$${struct}"' struct {[^}]\{1,\}}~~' | tr '\r' '\n' >"$${f}.tmp"; \
+	        mv -f "$${f}.tmp" "$${f}"; \
+	      fi; \
+	    fi \
+	  done \
+	done
+lint:
```

Comment:
This is not being used, but I think we can keep it around for formatting.

Even though the example is running, I may still have a problem with the path and module declarations, `make lint` says:

```
WARN [runner] Can't run linter goimports: goimports: analysis skipped: errors in package: [-: main module (github.com/t0mk/gometal) does not contain package github.com/t0mk/gometal/v1] 
ERRO Running error: goimports: analysis skipped: errors in package: [-: main module (github.com/t0mk/gometal) does not contain package github.com/t0mk/gometal/v1] 
```